src/demos/Welcome/main.vala

```diff
@@ -0,0 +1,11 @@
+Gtk.Builder builder;
+
+public void set_builder (Gtk.Builder b) {
+	builder = b;
+}
+
+public void run () {
+	print("Hello World from Vala code!\n");
+	var label = builder.get_object ("text") as Gtk.Label;
+	//label.label = "Hi!";
+}
```

Comment:
is it possible to pass an argument to run?

if so something like this would look great

```suggestion
public void run (workbench: Workbench) {
	print("Hello World from Vala code!\n");
	var label = workbench.builder.get_object ("text") as Gtk.Label;
	//label.label = "Hi!";
}
```

Reply:
I think that cannot work, because run() is a dbus method and how would you pass a reference to workbench over dbus?

Reply:
workbench here wouldn't be the JavasScript object - it would be a Vala object with `builder` assigned to the Vala builder and `Window` to the vala winow

The idea is just to have the same "API" as JavaScript